runtime/executor/method.cpp

```diff
@@ -1088,7 +1088,9 @@ Method::set_input(const EValue& input_evalue, size_t input_idx) {
     return Error::InvalidArgument;
   }
 
-  if (e.isTensor()) {
+ if (e.isNone()) {
```

Comment:
What are you doing here?

Reply:
Oh you are avoiding the error in the else.

Can you 1. Check this after tensor since tensor will be the common case. 2. Use else if

Reply:
Had to use "else if" indeed